sw/airborne/modules/geo_mag/geo_mag.c

```diff
@@ -49,6 +51,7 @@ void geo_mag_init(void)
 
 void geo_mag_periodic(void)
 {
+  //FIXME: kill_throttle has no place  in a geomag module
```

Comment:
This was a deliberate "feature" so that it doesn't calc the data while you are already in flight (as on the older boards it can take quite a while and will delay the control loops during that time...)